addon/components/justa-table.js

```diff
@@ -22,6 +22,34 @@ export default Component.extend({
     }
   },
 
+  didInsertElement() {
+    this._super(...arguments);
+    this._uninstallStickyHeaders();
+    Ember.run.scheduleOnce('afterRender', this, this._installStickyHeaders);
```

Comment:
Without using scheduleOnce, there were display issues with the columns changing in size when scrolling below the header.